src/main/java/dao/H2Factory/utils/CustomerMapper.java

```diff
@@ -0,0 +1,19 @@
+package dao.H2Factory.utils;
+
+import customerproductorder.models.Customer;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.springframework.jdbc.core.RowMapper;
+
+public class CustomerMapper implements RowMapper {
```

Comment:
You can annotate with @Component and inject this mapper in DAO with 

@Resource(name="customerMapper") RowMapper mapper;